src/pages/Login/Signup.js

```diff
@@ -21,30 +21,23 @@ class Login extends Component {
 			number: "",
 			password: "",
 			confirmPassword:"",
+			passwordMatch: true,
 			isHidden: true,
 	    };
 	  }
 	  
-	toggleHidden () {
+	toggleHidden = () => {
 		const {isHidden}=this.state;
 		this.setState({
 			isHidden: !isHidden
 		})
 	}	
 
-  	verifiedEmail(){
-  		const {email}=this.state;
-  		if(!email.includes("@") )return false
-  		else if(!email.split("@")[1].includes(".")) return false
-  		return true
-  	}
-	validateForm() {
-		const {name, email, number, password, confirmPassword}= this.state;
-		if(name==="" || email==="" || number === ""|| password==="" || confirmPassword==="") return false
-		else if(password!==confirmPassword) return false
-		else if(!this.verifiedEmail) return false
-		return true;
-	}
+	passwordLengthError = () => {
+		if (this.state.password.length < 6) {
+			alert("Password is not at least 6 characters");
```

Comment:
Same thing I mentioned above.

Reply:
Sorry accidently pressed resolved.
Is there a way to add a screenshot on here?